docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md

```diff
@@ -412,7 +412,10 @@ Typically, you can add a `gw` to the hostname and use nslookup to find it.
 You can also use the [Platform.sh CLI command](https://docs.platform.sh/development/cli.html) to find [the endpoint](https://docs.platform.sh/domains/steps/dns.html?#where-should-the-cname-point-to):
 
 ```bash
-    $ platform environment:info edge_hostname
+    # Define ibexa_cloud alias if it not already exists:
+    alias ibexa_cloud='PLATFORMSH_CLI_API_URL=https://api.cloud.ibexa.co PLATFORMSH_CLI_SESSION_ID=ibexa_cloud platform'
+    # Get the endpoint:
+    ibexa_cloud environment:info edge_hostname
```

Comment:
The alias is not needed for `environment:` commands. But we should be consistent and to use the alias `ibexa_cloud` everywhere instead of `platform` is what I recommend.

The alias can't be used out of nowhere so I redefine it. Maybe it could be introduced in the previous sentence by using a link to getting_started/install_on_ibexa_cloud instead of [docs.platform.sh/development/cli.html](https://docs.platform.sh/development/cli.html).